THCalendarDatePickerExample/THCalendarDatePickerExample/TestViewController.m

```diff
@@ -46,6 +45,7 @@ - (IBAction)touchedButton:(id)sender {
     [self.datePicker setSelectedBackgroundColor:[UIColor colorWithRed:125/255.0 green:208/255.0 blue:0/255.0 alpha:1.0]];
     [self.datePicker setCurrentDateColor:[UIColor colorWithRed:242/255.0 green:121/255.0 blue:53/255.0 alpha:1.0]];
     [self.datePicker setCurrentDateColorSelected:[UIColor yellowColor]];
+    self.datePicker.date = self.curDate;
```

Comment:
If you move it to the bottom u'll override some settings... Making them useless
Just have a look at the setDate code.

> On 31 Mar 2016, at 21:13, vkaramov <notifications@github.com> wrote:
> 
> In THCalendarDatePickerExample/THCalendarDatePickerExample/TestViewController.m:
> 
> > @@ -46,6 +45,7 @@ - (IBAction)touchedButton:(id)sender {
> >      [self.datePicker setSelectedBackgroundColor:[UIColor colorWithRed:125/255.0 green:208/255.0 blue:0/255.0 alpha:1.0]];
> >      [self.datePicker setCurrentDateColor:[UIColor colorWithRed:242/255.0 green:121/255.0 blue:53/255.0 alpha:1.0]];
> >      [self.datePicker setCurrentDateColorSelected:[UIColor yellowColor]];
> > +    self.datePicker.date = self.curDate;
> What setting? I've moved setting of the date property to the end to make it work properly.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
>